src/renderer.ts

```diff
@@ -29,9 +29,32 @@
 import { ProxyData } from './types'
 import './index.tsx'
 import { setMonacoEnv } from './components/Monaco/setMonacoEnv'
+import * as Sentry from '@sentry/electron/renderer'
 
 setMonacoEnv()
 
+// Sentry integration
+if (process.env.NODE_ENV !== 'development') {
+  Sentry.init({
+    integrations: [
+      Sentry.browserTracingIntegration(),
+      Sentry.replayIntegration(),
+    ],
+    tracesSampleRate: 1.0,
+    replaysSessionSampleRate: 0.1,
+    replaysOnErrorSampleRate: 1.0,
```

Comment:
Enabled some basic integrations such as browser tracing and session replay, which may help us debug issues when they arise.

Reply:
Should we consider filtering events to scrub sensitive data?